python/ray/actor.py

```diff
@@ -271,7 +272,21 @@ def _bind(
             options,
             other_args_to_resolve=other_args_to_resolve,
         )
-        return node
+
+        if node.num_returns > 1:
```

Comment:
Why do we need to handle `num_returns > 1` and `num_returns == 1` differently? Can we unify the logic? That is, if `num_returns == 1`, then `len(output_nodes)` should also be 1.

Reply:
Ack.

Reply:
Done.